static/src/stylesheets/module/experiments/_acquisitions-epic-testimonials.scss

```diff
@@ -0,0 +1,81 @@
+@import 'svg';
+
+// CSS used for the Epic testimonials test
+
+// Common classes
+
+.epic__testimonial__container {
+    margin-top: 20px;
```

Comment:
Could you try to convert most of the `margin`s pixel values into `$gs-baseline`?